src/redux/auth/saga.ts

```diff
@@ -79,6 +78,7 @@ export function* signupSaga({
   }
 }
 
+// TO FIX: still log in with  Fb in iOS by Facebook App doesn't work (not redirecting back to app)
```

Comment:
Can we use also `"TODO:"` tag here? Or we can add a new standardized tag, like `"FIXME:"`. I'm not sure, let me know your opinion :smile: